change/@ni-nimble-components-df572d28-5858-4799-8684-fabcd595dfab.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Think there are open questions @fredvisser and @jattasNI were following up on:

1. PO requirements on if  both decimal vs binary display of byte sizes, do we need both or can we standardize on one?
2. If showing binary display, can we use the NIST / ISO / ANSI [binary prefixes](https://en.wikipedia.org/wiki/Binary_prefix), i.e. MiB, GiB and migrate all usages in SystemLink to be standardized (other pipes, etc) 
3. Do we have tech writing / ni.com / other guidance on decimal vs binary sizing we can align on to drive consistency across all applications?

Mert is OOO this week and with the open questions I think we are blocked on reviews for now so moving back to draft. Feel free to re-open if there are parts to review ATM.